doc/dev_guide/release_management.rst

```diff
@@ -10,92 +10,134 @@ Release management
+   After the tag is set, no new changes are allowed to the release branch,
+   and it is declared deprecated and superseded by a newer MINOR version."
+
+Like Ubuntu, we distinguish between two kinds of stable releases:
+
+* **LTS (Long Term Support)** is a release series that is supported
+  for 3 years (community) and up to 5 years (paying customers).
+  Current LTS release series is 1.10.
 
-* The first digit stands for MAJOR release. A **major** release may contain
-  *incompatible changes*.
-* The second digit stands for MINOR release, it does not contain incompatible
-  changes, and is used for introducing backward-compatible *features*.
-* The third digit is for PATCH releases that contain only backward-compatible
-  *bug fixes*.
-
-In MINOR digit, we reflect how stable a release is:
-
-* 0 meaning alpha,
-* 1 meaning beta,
-* anything between 1 and 10 meaning stable, and
-* 10 meaning LTS.
-
-So, each MAJOR release series goes through a development-maturity life cycle of
-MINOR releases, as follows:
-
-1. **Alpha**. Once in every few months we release a few alpha versions,
-   e.g. 2.0.1, 2.0.2.
-
-   Alpha versions may contain incompatible changes, crashes and other bugs.
-
-2. **Beta**. Once major changes necessary to introduce new flagship features
-   are ready, we release a few beta versions, e.g. 2.1.3, 2.1.4.
-
-   Beta versions may contain crashes, but do not have incompatible changes,
-   so can be used to develop new applications.
-
-4. **Stable**. Finally, after we see our beta versions run successfully in
-   production, usually in a few more months, during which we fix all incoming
-   bugs and add some minor features, we declare this MAJOR release series
-   stable.
-
-Like Ubuntu, we distinguish two kinds of stable releases:
-
-* **LTS (Long Term Support)** releases that are supported for 3 years
-  (community) and up to 5 years (paying customers). **LTS** release
-  is identified by MINOR version 10.
 * **Standard stable releases** are only supported a few months after the next
   stable is out.
 
 "Support" means that we continue fixing bugs in a release.
 
-We add commits simultaneously to three MAJOR releases:
+We add bug fixes simultaneously into the following three releases and
+into the current alpha release:
 
-* **LTS** is a stable release which does not receive new features, and only gets
-  backward compatible fixes. Hence, following the rules of semver, LTS release
+* **LTS** is a stable release which does not receive new features and only gets
+  backward compatible fixes. Hence, LTS release
   never has its MAJOR or MINOR version increased, and only gets PATCH level
   releases.
 
-* **STABLE** is our current stable release, which may receive new features.
+* **STABLE** are our current stable releases. They also receive no new features
+  and only backward compatible fixes.
   When the next STABLE version is published, MINOR version is incremented.
```

Comment:
> When the next STABLE version is published, MINOR version is incremented.

It is true if we tell about a stable release series. Otherwise we should say about PATCH version increment.

In fact this sentence repeats the information above and I don't see much sense to keep it here.